src/Cimpress.Nancy.Services/ContentTypeVersionModuleExtender.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Cimpress.Nancy.Components;
+using System.Collections.Generic;
+using System.Linq;
+using Nancy;
+
+namespace Cimpress.Nancy.Services
+{
+    public class ContentTypeVersionModuleExtender : IVersionModuleExtender
+    {
+        private readonly IEnumerable<string> _entityMethods = new List<string> {"put", "post", "patch"};
+
+        public void After(NancyContext context)
+        {
+
+        }
+
+        public Response Before(NancyContext context)
+        {
+            var methodType = context.Request.Method.ToLower();
+            if (_entityMethods.Contains(methodType))
+            {
+                return new Response
+                {
+                    StatusCode = HttpStatusCode.BadRequest,
+                    ReasonPhrase = "Content-Type Header not supplied"
```

Comment:
Where do you actually check if the Content-Type header is null?

Also, what else do you plan on putting in the project to warrant its existence? 
If you do have plans for more, maybe the user should have a way of selecting which of these "Services" to enable?
Maybe an optional parameter in the IConfiguration object to enable it (or disable it).